proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -75,3 +76,18 @@ message MintAction {
   bytes to = 1;
   astria.primitive.v1.Uint128 amount = 2;
 }
+
+message Ics20Withdrawal {
+  astria.primitive.v1.Uint128 amount = 1;
+  string denom = 2;
+  string destination_chain_address = 3;
+  bytes return_address = 4;
+  IbcHeight timeout_height = 5;
+  uint64 timeout_time = 6;
```

Comment:
units? greater 0?

Reply:
copied the comments on the rust type here - this is a unix timestamp